### PR TITLE
[CHORE] Subscribe to permissions

### PR DESCRIPTION
**Pull request by kresnaputra**
## Proposed changes

If a permissions is changed by another user or on another client, the permissions's local database isn't change right away. So we need to subscribe or stream the permissions for update the local database every time permissions is change

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by gerzonc — APPROVED** [review]
LGTM

File: app/sagas/login.js [hunk 1]
```diff
@@ -87,6 +87,7 @@ const handleLoginRequest = function* handleLoginRequest({ credentials, logoutOnE
 };
 
 const fetchPermissions = function* fetchPermissions() {
+	RocketChat.subscribe('stream-notify-logged', 'permissions-changed');
```
Comment (diegolmello) [inline app/sagas/login.js:90]: Can you do it inside `RocketChat.getPermissions()`?

File: app/actions/permissions.js [hunk 2]
```diff
@@ -6,3 +6,10 @@ export function setPermissions(permissions) {
 		permissions
 	};
 }
+
+export function updatePermissions(id, roles) {
```
Comment (diegolmello) [inline app/actions/permissions.js:10]: Plural, but updates a single permission.

**Review by diegolmello — CHANGES_REQUESTED** [review]

**Review by gerzonc — APPROVED** [review]
LGTM

**Pull request merged**